configuration.go

```diff
@@ -5,13 +5,16 @@ import (
 )
 
 type ConsumerConfiguration struct {
-	KeyDeserializer   string `json:"keyDeserializer"`
-	ValueDeserializer string `json:"valueDeserializer"`
+	KeyDeserializer     string `json:"keyDeserializer"`
+	ValueDeserializer   string `json:"valueDeserializer"`
+	SubjectNameStrategy string `json:"subjectNameStrategy"`
+	UseMagicPrefix      bool   `json:"useMagicPrefix"`
```

Comment:
I suppose this will be removed when we implement #50 and #55.

Reply:
Yes, since these are serde configurations.